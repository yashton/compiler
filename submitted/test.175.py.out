(ID "x")
(PUNCT "=")
(ERROR "Illegal character '\"\"\"\"\"\"' on line 1)
