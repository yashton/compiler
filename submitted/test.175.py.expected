(ID "x")
(PUNCT "=")
(ERROR "unknown char: '\'")
(ERROR "unterminated string literal at EOF")
